pwiz_tools/Skyline/Properties/Resources.Designer.cs

```diff
@@ -365,7 +365,7 @@ public class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to An error occurred while trying to display the document &apos;{0}&apos;. 
+        ///   Looks up a localized string similar to An error occurred while trying to display the document &apos;{0}&apos;. 
```

Comment:
Seems like a lot of unintended changes. Are your line break settings causing these?